checkbox-ng/checkbox_ng/urwid_ui.py

```diff
@@ -691,11 +691,11 @@ def interrupt_dialog(host):
         ('start', 'dark green,bold', 'black'),
     ]
     choices = [
-        _("Cancel the interruption and resume the session (ESC)"),
-        _("Kill test in progress and move on to next"),
-        _("Disconnect the master (Same as CTRL+C)"),
-        _("Stop the checkbox slave @{}".format(host)),
-        _("Abandon the session on the slave @{}".format(host)),
+        _("Nothing, continue testing (ESC)"),
+        _("Kill test in progress and move on to the next"),
+        _("Exit but let the service test run continue (CTRL+C)"),
+        _("Exit and kill the service test runer at {}".format(host)),
```

Comment:
Maybe `Exit and kill the Checkbox service on the testbed at...`?

Reply:
s/kill/stop.
But I think we should remove this option altogether.
In 99%+ cases this would mean that the service is just restarted.